packages/FormElement/stories/FormElement.stories.mdx

```diff
@@ -102,160 +97,55 @@ The FormElement can be used in a variety of formats. These can include FormEleme
 
 ##### Result
 
-<CheckboxExample />
-
-#### Radios
-
-```jsx
-<FormElement hasFieldSet label="Form Label">
-  <FormElement.Content>
-    {({ ariaDescribedBy: ariaDescribedByOuter }) => getRadioOptions(ariaDescribedByOuter)}
-  </FormElement.Content>
-  <FormElement.Description>
-    <span>Description of this field.</span>
-  </FormElement.Description>
-  <FormElement.Error>{errorText}</FormElement.Error>
-  <FormElement.Help>
-    Give me some help. <a href="wegalvanize.com">Learn more</a>.
-  </FormElement.Help>
-</FormElement>
-```
-
-##### Result
+<FormElementStory>
+  <Tagline>Form Element with checkboxes.</Tagline>
+  <br />
+  <CheckboxExample />
+</FormElementStory>
+<Rule />
+<FormElementStory>
+  <Tagline>Form Element with radios.</Tagline>
+  <br />
+  <RadioExample />
+</FormElementStory>
```

Comment:
I have a few more ideas about this, but I think it's better to discuss together.